### PR TITLE
Added opening PDFs from URL

### PR DESCRIPTION
**Pull request by darkcheftar**
Addressing #25 I used cors-anywhere and hosted it at Heroku at [This Proxy](https://pdf-to-markdown-proxy.herokuapp.com/) link following [This tutorial](https://youtu.be/zoOx1b9iBRk). And taking the URL to be opened using prompt and opening the URL using the above proxy. 
Taking link using prompt:
image
Opening pdf:
image

### CODE REVIEW

File: ui/src/main/Upload.svelte [hunk 1]
```diff
@@ -13,6 +13,14 @@
     let upload: Promise<any>;
     let rejectionError: string;
     let parseProgress: Progress;
+    function handleUrlLoad() {
+        dragover = true;
+        var answer = prompt('Url of the pdf');
```
Comment (jzillmann) [inline ui/src/main/Upload.svelte:18]: I would just make this a `let` or `const`...
Also at some point I might try to convert this to a homegrown popup using the `Popup` component, but that isn't crucial!

File: ui/package.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "@fortawesome/free-solid-svg-icons": "^5.15.2",
+    "autoprefixer": "^10.2.5",
```
Comment (jzillmann) [inline ui/package.json:24]: Any reason you added this ?

**Review by jzillmann — COMMENTED** [review]
So yeah, looks good to me! I would prefer removing the `autoprefixer` dependency (unless there is a good reason not to do so) and then it is ready to go!

Btw, was there a specific reason you spawn up your own proxy instead of using an existing on e ?

File: ui/package.json (hunk 2 above)
Comment (darkcheftar) [reply ui/package.json:24]: Nope! It was unintended I have not used it. I will remove it asap!

File: ui/src/main/Upload.svelte (hunk 1 above)
Comment (darkcheftar) [reply ui/src/main/Upload.svelte:18]: Nope, no specific reason to spawn a new proxy. I haven't found any reliable proxy. Hence I made one for testing. 
Please suggest to me if you find any existing reliable proxy.

**Comment by darkcheftar** [comment]
I cleared the minor inconveniences
- using `let` keyword instead of `var`
- removing `autoprefixer` from dependencies

**Comment by jzillmann** [comment]
Ok, I was curious about the large changes of `package-lock.json` and played a bit with it. Seems like the newest version of `pdfjs` broke something with the worker setup so I pinned all production libs for now!

**Review by jzillmann — APPROVED** [review]

**Comment by jzillmann** [comment]
Merged, thanks for digging in!

**Comment by darkcheftar** [comment]
> Ok, I was curious about the large changes of `package-lock.json` and played a bit with it and run in trouble. Seems like the newest version of `pdfjs` broke something with the worker setup so I pinned all production libs for now!

I didn't noticed that.
Can I do something about it?
Any suggestions?

**Comment by jzillmann** [comment]
👆 if you want you could investigate. Just upgrade to the latest pdfjs and see if it works.

**Comment by darkcheftar** [comment]
> 👆 if you want you could investigate. Just upgrade to the latest pdfjs and see if it works.
sure, I take a look into it sire.

**Comment by darkcheftar** [comment]
> 👆 if you want you could investigate. Just upgrade to the latest pdfjs and see if it works.

sire @jzillmann , actually I wanted to do this but I didn't understood what to do exactly.
do I just need to update pdfjs or do I need to run `npm update` or something.
could you please explain what to do in detail if possible?

**Comment by jzillmann** [comment]
So since this is totally different issue I created #28 for it!
I put more details into it. If somethings missing, let me know!

**Comment by darkcheftar** [comment]
> So since this is totally different issue I created #28 for it!
> I put more details into it. If somethings missing, let me know!

sure, sire

**Pull request merged**